### PR TITLE
user feedback on voting

### PR DESCRIPTION
**Pull request by turadg**
Fixes: #13 

Using https://fkhadra.github.io/react-toastify/promise/#toastpromise would be a little cleaner but not sure it's worth the effort

### CODE REVIEW

File: src/utils/displayFunctions.tsx [hunk 1]
```diff
@@ -8,9 +8,35 @@ import { IST_ICON } from 'assets/assets';
...
-export const notifyError = err => {
+import { Id as ToastId, toast } from 'react-toastify';
+import { DeliverTxResponse } from '@cosmjs/stargate';
+
+export const notifySigning = () =>
+  toast.loading(<p>Awaiting sign and broadcast…</p>);
+
+export const notifySuccess = (toastId: ToastId, tx: DeliverTxResponse) => {
+  const txUrl = `https://bigdipper.live/agoric/transactions/${tx.transactionHash}`;
+  toast.update(toastId, {
```
Comment (samsiegart) [inline src/utils/displayFunctions.tsx:19]: We should make the success and error toasts dismissible

File: src/utils/displayFunctions.tsx [hunk 2]
```diff
@@ -8,9 +8,35 @@ import { IST_ICON } from 'assets/assets';
...
+    render: (
+      <p>
+        <a href={txUrl} target={tx.transactionHash}>
+          Transaction
+        </a>{' '}
+        complete.
```
Comment (samsiegart) [inline src/utils/displayFunctions.tsx:25]: I think this should say "sent" instead of "complete"

**Comment by samsiegart** [comment]
I can rebase https://github.com/Agoric/dapp-econ-gov/pull/16 after this goes in. Just left a couple comments.

**Comment by dckc** [comment]
passed a smoke-test here.

The explorer link goes to mainnet, but with some url surgery, [my ollinet tx](https://ollinet.explorer.agoric.net/agoric/tx/CFD7DF0C67B119FD25D86B4C0C7697A151BF66552EB967128BAE840E2E04485C) is easy enough to find.

image

File: src/utils/displayFunctions.tsx (hunk 1 above)
Comment (otoole-brendan) [reply src/utils/displayFunctions.tsx:19]: agreed

**Comment by otoole-brendan** [comment]
Same as Dan - smoke tested and works great (if we can just point to right explorer)

**Comment by dckc** [comment]
Maybe if the network isn't mainnet, put the tx in a title attribute instead of a link.

**Comment by otoole-brendan** [comment]
My point about wrong explorer is moot, happy to approve

**Review by otoole-brendan — APPROVED** [review]
LGTM

**Pull request merged**